finalPrep/modZeroHero.js

```diff
@@ -2,52 +2,116 @@
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ['dishes', ' bedtime', ' clear legos on the carpet'];
+var sidekicks = ['Will', 'Hazel', 'Hank'];
+
 // Print the first sidekick to your console
 
+console.log('sidekick #1', sidekicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log('last archenemy', archEnemies[2]);
+
 // Write some code to add a new archEnemy to the archEnemies array
 
+archEnemies = [...archEnemies, ' the cat box'];
+
 // Print the archEnemies array to console to ensure you added a new archEnemy
 
+console.log('archEnemies + new entry', archEnemies);
+
 // Remove the first sidekick from the sidekicks array
 
+sidekicks.shift();
+
 // Print the sidekicks array to console to ensure you added a new sidekick
 
+console.log('sidekicks - first entry', sidekicks);
+
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
+function assessSituation(dangerLevel) {
+  this.dangerLevel = dangerLevel
+  var saveTheDay = 'Here we go again!'
+  var badExcuse = 'I think I left the toaster on.'
+  if (dangerLevel >= 50) {
+    console.log(badExcuse)
+  } else if (dangerLevel < 10) {
+      console.log('Meh. Hard pass.')
+    } else {
+      console.log(saveTheDay)
+    }
+};
+
+assessSituation(70);
+assessSituation(50);
+assessSituation(40);
+assessSituation(5);
+
 //Test Cases
 var announcement = 'Never fear, the Courageous Curly Bracket is here!';
 var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.';
 // assessSituation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
```

Comment:
Lines 94 - 96 are calling the function `assessSituation` passing in **three** arguments.